client/src/actions/createTicket.js

```diff
@@ -0,0 +1,6 @@
+export const createTicket = (Ticket) => {
```

Comment:
Why is there a separate action creator here? Also creating a new ticket has nothing to do with this PR.